mt940/models.py

```diff
@@ -381,7 +381,7 @@ def parse(self, data):
                     self.transactions.append(transaction)
                 else:
                     transaction.data.update(result)
-            elif tag.scope is Transaction and self.transactions:
```

Comment:
Hmm, I see. In that case it might be best to simply create a new class that subclasses both `Transaction` and `Transactions`. And... change the code to use `if` instead of `elif`